source/elements/oneDNN/source/graph/ops/index.rst

```diff
@@ -0,0 +1,113 @@
+.. SPDX-FileCopyrightText: 2020-2022 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+#############
+Operation Set
+#############
+
+oneDNN Graph defines an operation set. oneDNN Graph implementation may support a
+subset of the operation set.
+
+.. toctree::
+
+   arithmetic/Abs_1.rst
+   arithmetic/AbsBackprop_1.rst
+   arithmetic/Add_1.rst
+   pooling/AvgPool_1.rst
+   pooling/AvgPoolBackprop_1.rst
+   normalization/BatchNormForwardTraining_1.rst
+   normalization/BatchNormInference_1.rst
+   normalization/BatchNormTrainingBackprop_1.rst
+   arithmetic/BiasAdd_1.rst
+   arithmetic/BiasAddBackprop_1.rst
+   activation/Clamp_1.rst
+   activation/ClampBackprop_1.rst
+   movement/Concat_1.rst
+   convolution/Convolution_1.rst
+   convolution/ConvolutionBackpropData_1.rst
+   convolution/ConvolutionBackpropWeights_1.rst
+   convolution/ConvTranspose_1.rst
+   convolution/ConvTransposeBackpropData_1.rst
+   convolution/ConvTransposeBackpropWeights_1.rst
+   lower_precision/Dequantize_1.rst
+   arithmetic/Divide_1.rst
+   lower_precision/DynamicDequantize_1.rst
+   lower_precision/DynamicQuantize_1.rst
+   shape/DynamicReshape_1.rst
+   movement/DynamicTranspose_1.rst
+   activation/Elu_1.rst
+   activation/EluBackprop_1.rst
+   misc/End_1.rst
+   arithmetic/Erf_1.rst
+   comparison/Equal_1.rst
+   activation/Exp_1.rst
+   activation/GELU_1.rst
+   activation/GELUBackprop_1.rst
+   comparison/Greater_1.rst   
```

Comment:
The comparison, condition, logical operations, and a few other operations were removed when we merged graph API into oneDNN master. I feel we need update here according to the op kinds defined in dnnl_graph.hpp and the documents in https://github.com/intel-innersource/libraries.performance.math.onednn/pull/4400.

Reply:
Makes sense, will remove those until we get a stable functional implementation.